### DISCUSSION TITLE
Bad blood

### DISCUSSION THREAD

**Discussion by amis92** [BSData/adeptus-titanicus#79] [General]
@capitaladot It's been brought to my attention there has been an issue with @Deviltree's contributions recently.

1. Your maintenance as of last year (excl. last week) of this repository can't be considered an active one. Thus, your semi-claim of ownership on this repository doesn't have a solid ground.
2. Your reaction to @Deviltree's self-imposed refactor was correct in terms of action taken (denial).
3. Your attitude on this action didn't adhere to our Code of Conduct.

As a reminder, our official Code of Conduct is hosted at https://github.com/BSData/catalogue-development/wiki/Code-of-Conduct. For reference, it says (bold for attention):

1. **Be polite.** Everyone here gets no money for this work. It's open-source community.
2. Don't complain. Fix it. Or at least open an issue. Someone probably will fix it.
3. Spend only spare time here. You won't get money for this.
4. Try to fix current issues first.

I don't think how you reacted to this situation can be considered polite. Please also bear in mind we want to be an open and welcoming community. It's always easy to be polite when things are cool. What really matters and has a considerable impact on wider community's reception of our organization, is how we handle those *bad* behaviors.

PS. Code of Conduct you've added to the README should at least state it's an extension of our main one and link to it. Honestly, it's not even a code of conduct you wrote, it's just a reactionary statement.

---

@Deviltree what you attempted can't be considered an action in good faith.
1. When coming in to a new repository, you should definitely try to contact maintainers on whether they will be open for large changes. *You did contact @TrentBartlem but you just sent him a "Crucible of Retribution" update.*
1. As for #70, your commit messages nor PR title didn't state that you did a huge refactor.

From what I've seen, you didn't really come forward with the removal of Structure Points. Yet you did consider this a major change in other statements. So it does look a bit like concealing your intentions, even if you weren't.

It's understandable that you wanted a better experience when using this repository. We also always appreciate people coming in, since it's a difficult ecosystem to work in (Data Editor, XML files). We really try to be a welcoming community. 

---

Now, I'd like for both of you to clear out the atmosphere and treat each other with respect, openness and politeness. Go ahead, I'll wait. I'll also discuss through with you anything you want. :)

**Comment by ghost** [comment]
Thanks for bringing the upstream code of conduct to my attention; I didn't know it existed.
I thanked @Deviltree for their contribution on gitter but spelled out my objections to how they behaved. 
I went through and hand-edited in what they provided.

**Comment by amis92** [comment]
> Thanks for bringing the upstream code of conduct to my attention; I didn't know it existed.

It's not easily discoverable. We should probably make that more prominent.

**Comment by Deviltree** [comment]
Hey Guys,
Sorry it took me so long to get back to you, I was with Family.   I'd like to apologize first of all, I'm not trying to stir anything up, I really just wanted to bring a repository that's had no meaningful updates in well over six months despite new rules and models being released.  When I reached out on the BS discord to find the person who managed the repo, Trent was the only person who responded, and I told him what I was up to.  I should have been more explict in removing structure as well, I'll do a better job of posting the back log moving forward.

I think we can all come up with some really simple guides for communicating here.  I very much want to keep BS up to date and am super excited that @Capitaladot put the Warmaster in.  I can merge an earlier build with the structure in it with the two new updates to resolve the current state of the repo, then we can hopefully all get together on BS or here and agree to a plan to keep the repository tight.  I'd love to have this repo as comprehensive and quickly updated as the 40k repo. 

Best,
Matt
